GameFiles/GameInterface/_integration_base.gsc

```diff
@@ -360,7 +361,7 @@ GenerateJoinTeamString( isSpectator )
         }
     }
 
-    guid = self GetXuid();
+    guid = self [[level.overrideMethods[level.commonFunctions.getXuid]]]();
```

Comment:
So after thinking about this, I think we should move the `GenerateJoinTeamString` into the _integration_shared.gsc 
I added this in here before I created the shared functionality file.
Philosophically the _integration_base,gsc should really only be responsible for facilitating the game interface data transfer/notifications. No direct game interaction or action on game events.